kratos/utilities/delaunator_utilities.cpp

```diff
@@ -0,0 +1,206 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+#define REAL double
```

Comment:
this cannot be defined globally...as a minimum it must be undefined at the end of the file...

Reply:
OK

Reply:
It cannot b e defined

Reply:
In any case this is not defined globally, this is a cpp file (not a header), an it is the only way to make triangle to work. All implementations of triangle work that way.

Reply:
you can do `#undef REAL` at the end of the file

Reply:
ok this is done